### PR TITLE
fixes BZ1411226.  Identifies the needed ports for aggregated logging

### PR DESCRIPTION
**Pull request by jcantrill**

### CODE REVIEW

File: install_config/install/prerequisites.adoc [hunk 1]
```diff
@@ -581,6 +581,21 @@ connections, and is only required if you have clustered etcd.
 | For use by the Kubelet. Required to be externally open on nodes.
 |===
 
+.Aggregated Logging
+[cols='2,1,8']
+|===
+| *9200*
+|TCP
+|For Elasticsearch API use.  Required to be internally open on any infrastrure
```
Comment (richm) [inline install_config/install/prerequisites.adoc]: infrastructure

File: install_config/install/prerequisites.adoc [hunk 2]
```diff
@@ -581,6 +581,21 @@ connections, and is only required if you have clustered etcd.
...
+nodes so the Kibana component is able to retrieve logs for display.  It can be
+open externally for direct access Elasticsearch.
```
Comment (richm) [inline install_config/install/prerequisites.adoc]: but that also requires a service (which is left as an exercise for the reader).

**Comment by ewolinetz** [comment]
@ahardin-rh

File: install_config/install/prerequisites.adoc [hunk 3]
```diff
@@ -581,6 +581,21 @@ connections, and is only required if you have clustered etcd.
...
+
+| *9300*
+|TCP
+|For Elasticsearch inter-cluster use.  Required to be internally open on any infrastrure
```
Comment (richm) [inline install_config/install/prerequisites.adoc]: infrastructure

File: install_config/install/prerequisites.adoc (hunk 1 above)
Comment (jcantrill) [inline install_config/install/prerequisites.adoc]: fixed

**Comment by jcantrill** [comment]
updated

**Comment by richm** [comment]
LGTM

File: install_config/install/prerequisites.adoc [hunk 4]
```diff
@@ -588,6 +588,22 @@ connections, and is only required if you have clustered etcd.
 * {product-title} internal DNS cannot be received over SDN. Depending on the detected values of `*openshift_facts*`, or if the `*openshift_ip*` and `*openshift_public_ip*` values are overridden, it will be the computed value of `*openshift_ip*`. For non-cloud deployments, this will default to the IP address associated with the default route on the master host. For cloud deployments, it will default to the IP address associated with the first internal interface as defined by the cloud metadata.
 * The master host uses port *10250* to reach the nodes and does not go over SDN. It depends on the target host of the deployment and uses the computed values of `*openshift_hostname*` and `*openshift_public_hostname*`.
 
+.Aggregated Logging
+[cols='2,1,8']
+|===
+| *9200*
+|TCP
+|For Elasticsearch API use.  Required to be internally open on any infrastructure
+nodes so the Kibana component is able to retrieve logs for display.  It can be
```
Comment (ewolinetz) [inline install_config/install/prerequisites.adoc]: so that the

File: install_config/install/prerequisites.adoc [hunk 5]
```diff
@@ -588,6 +588,22 @@ connections, and is only required if you have clustered etcd.
...
+open externally for direct access to Elasticsearch and used with a service added
```
Comment (ewolinetz) [inline install_config/install/prerequisites.adoc]: externally opened for direct access to Elasticsearch by means of a route.  The route would need to be created

Comment (ewolinetz) [inline install_config/install/prerequisites.adoc]: we already create a service for port 9200, I think you meant "route" here

File: install_config/install/prerequisites.adoc [hunk 6]
```diff
@@ -588,6 +588,22 @@ connections, and is only required if you have clustered etcd.
...
+by a cluster administrator.
```
Comment (ewolinetz) [inline install_config/install/prerequisites.adoc]: Should we describe how to do this? Or at least reference `oc expose`

File: install_config/install/prerequisites.adoc (hunk 4 above)
Comment (ewolinetz) [inline install_config/install/prerequisites.adoc]: probably leave off "the" and "component" -- so that Kibana is able to retrieve...

**Comment by ahardin-rh** [comment]
My only nit is to have only 1 space between sentences. Also, is this for 3.3 and beyond?

File: install_config/install/prerequisites.adoc (hunk 4 above)
Comment (jcantrill) [inline install_config/install/prerequisites.adoc]: updated.

File: install_config/install/prerequisites.adoc (hunk 5 above)
Comment (jcantrill) [inline install_config/install/prerequisites.adoc]: removed ref to service.

File: install_config/install/prerequisites.adoc (hunk 6 above)
Comment (jcantrill) [inline install_config/install/prerequisites.adoc]: Added comment regarding oc expose

**Comment by jcantrill** [comment]
Updated based on feedback.  These changes are relevant for logging from its inception.

**Comment by ewolinetz** [comment]
LGTM

**Comment by ahardin-rh** [comment]
Thanks!

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../install_config/install/prerequisites.adoc#install-config-install-prerequisites[Installing a Cluster -> Prerequisites]
|Added information about xref:../install_config/install/prerequisites.adoc#required-ports[required ports] for Aggregated Logging.
%

**Pull request merged**